content/install_maintain/installation/installing-cluster.md

```diff
@@ -370,198 +462,214 @@ Configure the following settings in `/etc/cloudify/config.yaml` for each Manager
+      networks:
+        default: <private ip of rabbit server 1>
+    <host name of rabbit server 2>:
+        networks:
+            default: <private ip of rabbit server 2>
+    <host name of rabbit server 3>:
+        networks:
+            default: <private ip of rabbit server 3>
+  monitoring:
+    username: monitoringusername
+    password: longyeteasytorememberstringasapassword
+
+
+postgresql_server:
+  ca_path: '/home/centos/.cloudify-test-ca/ca.crt'
+  postgres_password: areallystrongandsecretpasswordforpostgres
+  cluster:
+    nodes:
+      <first postgresql instance-name>:
+        ip: <private ip of postgres server 1>
+      <second postgresql instance-name>:
+        ip: <private ip of postgres server 2>
+      <third postgresql instance-name>:
+        ip: <private ip of postgres server 3>
+     
+
+postgresql_client:
+  ssl_enabled: true
+    
+  # Same password as the one of the PostgreSQL server.
+  # THE PASSWORD WILL BE REMOVED FROM THE FILE AFTER THE INSTALLATION FINISHES
+  server_password: 'areallystrongandsecretpasswordforpostgres'
+    
+  # If true, client SSL certificates will need to be supplied for database connections
+  ssl_client_verification: true
+  
+  monitoring:
+    username: monitoringusername
+    password: longyeteasytorememberstringasapassword
+
+# In case you use a load-balancer, you would need to specify its private IP
+# in order for the different agents to connect to it.
+networks:
+    load-balancer: <load-balancer private IP address>
+
+ssl_inputs:
+  internal_cert_path: '/home/centos/.cloudify-test-ca/<ip of this host>.crt'
+  internal_key_path: '/home/centos/.cloudify-test-ca/<ip of this host>.key'
+  external_cert_path: '/home/centos/.cloudify-test-ca/<ip of this host>.crt'
+  external_key_path: '/home/centos/.cloudify-test-ca/<ip of this host>.key'
+  ca_cert_path:  '/home/centos/.cloudify-test-ca/ca.crt'
+  external_ca_cert_path: '/home/centos/.cloudify-test-ca/ca.crt'
+  
+  #If you set 'ssl_client_verification' under 'postgresql_client' to true
+  postgresql_client_cert_path: '/home/centos/.cloudify-test-ca/<ip of this host>.crt'
+  postgresql_client_key_path: '/home/centos/.cloudify-test-ca/<ip of this host>.key'
+
+# For monitoring service(status reporter)
+prometheus:
+  blackbox_exporter:
+    ca_cert_path: '/home/centos/.cloudify-test-ca/ca.crt'
+  credentials:
+    username: 'monitoringusername'
+    password: 'longyeteasytorememberstringasapassword'
+
+  cert_path: <certificate for prometheus, cert_path of rabbitmq can be used>
+  key_path: <key for promethus, key_path of rabbitmq can be used>
+  ca_path: <ca for promethus, key_path of rabbitmq can be used>'
+
+services_to_install:
+    - manager_service
+    - monitoring_service
+
+```
+
+
+### File permissions
+
+After installing all nodes, make sure cfyuser is able to read certs you are using on managers by performing:
+
+`sudo chmod 0755 /home/centos && sudo chown cfyuser:cfyuser -R /home/centos/.cloudify-test-ca/`
```

Comment:
This:
a) Shouldn't be needed (because the certs get copied to their destinations)
b) Is a really bad idea, as it allows full read access to the user's home directory and (depending on umask) fully exposes the certs AND KEYS to anyone with any access to the system.

Please test it with this step completely removed. If it doesn't work at that point, it's a bug.

Reply:
it worked for me when the certs were in /tmp